File: ruby/problem_solving/codewars/8kyu/dna_to_rna_conversion.rb
# https://www.codewars.com/kata/5556282156230d0e5e000089/train/ruby

def dna_to_rna(dna)
  dna.gsub('T', 'U')
end

p dna_to_rna("TTTT") == "UUUU"
p dna_to_rna("GCAT") == "GCAU"
p dna_to_rna("GACCGCCGCC") == "GACCGCCGCC"

File: go/problem_solving/codewars/8kyu/dna_to_rna_conversion.go
package main

// https://www.codewars.com/kata/5556282156230d0e5e000089/train/go

import (
	"fmt"
	"strings"
)

func DNAtoRNA(dna string) string {
	return strings.ReplaceAll(dna, "T", "U")
}

func main() {
	fmt.Println(DNAtoRNA("TTTT") == "UUUU")
	fmt.Println(DNAtoRNA("GCAT") == "GCAU")
	fmt.Println(DNAtoRNA("GACCGCCGCC") == "GACCGCCGCC")
}
